Makefile

```
@@ -7,11 +7,27 @@ VCS_BRANCH := $(strip $(shell git rev-parse --abbrev-ref HEAD))
 VCS_REF := $(strip $(shell [ -d .git ] && git rev-parse --short HEAD))
 DOCKER_REPO ?= quay.io/observatorium/observatorium-operator
 
-BIN_DIR ?= $(shell pwd)/tmp/bin
+TMP_DIR := $(shell pwd)/tmp
+BIN_DIR ?= $(TMP_DIR)/bin
+CERT_DIR ?= $(TMP_DIR)/certs
 
 CONTROLLER_GEN ?= $(BIN_DIR)/controller-gen
 JB ?= $(BIN_DIR)/jb
+ GENERATE_TLS_CERT ?= $(BIN_DIR)/generate-tls-cert
 
+$(BIN_DIR):
+	mkdir -p $(BIN_DIR)
+
+$(CERT_DIR):
+	mkdir -p $(CERT_DIR)
+
+# Generate TLS certificates for local development.
+generate-cert: $(GENERATE_TLS_CERT) $(CERT_DIR)
```

We should make the dependency on the cert dir target order-only
```suggestion
generate-cert: $(GENERATE_TLS_CERT) | $(CERT_DIR)
```